core/domain/feedback_services.py

```diff
@@ -379,14 +381,26 @@ def create_messages(
     suggestion_models.GeneralSuggestionModel.put_multi(
         suggestion_models_to_update)
 
-    if (feconf.CAN_SEND_EMAILS and (
-            feconf.CAN_SEND_FEEDBACK_MESSAGE_EMAILS and
+    server_can_send_emails = (
+        platform_parameter_services.get_platform_parameter_value(
+            platform_parameter_list.ParamName.SERVER_CAN_SEND_EMAILS.value
+        )
+    )
+    # TODO(#12079): Figure out a better way to avoid sending feedback
+    # thread emails for contributor dashboard suggestions.
+    message_changed = (
+        len(text) > 0 or old_statuses[index] != new_statuses[index]
+    )
+    if (
+        server_can_send_emails and
+        (
```

Comment:
Os this bracket needed? And si used everywhere, thus "a and (b and c) and d" equals "a and b and c and d"